src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -81,4 +82,72 @@ public void testGetLastTradedPriceEmpty() throws Exception {
         .andReturn();
   }
+
+  @Test
+  public void testGetHistoricalPricesBySymbol() throws Exception {
+
+    MvcResult result = this.mvc.perform(
+            org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                .get("/iex/historicalPrices?symbol=ibm&token=pk_05d61965a1eb493a82af2953cfdb01ba")
+                // This URL will be hit by the MockMvc client. The result is configured in the file
+                // src/test/resources/wiremock/mappings/mapping-historicalPricesBySymbol.json
+                .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$", hasSize(21)))
+        .andExpect(jsonPath("$[0].symbol", is("IBM")))
+        .andExpect(jsonPath("$[0].close").value(new BigDecimal("138.56")))
+        .andExpect(jsonPath("$[0].date").value("2021-09-27"))
+        .andExpect(jsonPath("$[-1].symbol", is("IBM")))
+        .andExpect(jsonPath("$[-1].close").value(new BigDecimal("127.64")))
+        .andExpect(jsonPath("$[-1].date").value("2021-10-25"))
+        .andReturn();
+  }
+
+
+  @Test
+  public void testGetHistoricalPrices() throws Exception {
+
+    MvcResult result = this.mvc.perform(
+            org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                .get("/iex/historicalPrices?symbol=fb&range=3m&token=pk_05d61965a1eb493a82af2953cfdb01ba")
+                // This URL will be hit by the MockMvc client. The result is configured in the file
+                // src/test/resources/wiremock/mappings/mapping-historicalPrices.json
+                .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$[0].symbol", is("FB")))
+        .andExpect(jsonPath("$[0].close").value(new BigDecimal("373.28")))
+        .andExpect(jsonPath("$[0].date").value("2021-07-28"))
+        .andExpect(jsonPath("$[-1].symbol", is("FB")))
+        .andExpect(jsonPath("$[-1].close").value(new BigDecimal("312.22")))
+        .andExpect(jsonPath("$[-1].date").value("2021-10-27"))
+        .andReturn();
+  }
+
+  @Test
+  public void testGetHistoricalPricesByDate() throws Exception {
+
+    MvcResult result = this.mvc.perform(
+            org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                .get("/iex/historicalPrices?symbol=A&range=date&date=20200420&token=pk_05d61965a1eb493a82af2953cfdb01ba")
+                // This URL will be hit by the MockMvc client. The result is configured in the file
+                // src/test/resources/wiremock/mappings/mapping-historicalPricesByDate.json
+                .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$.length()").value(is(1)))
+        .andExpect(jsonPath("$[0].date").value("2020-04-20"))
+        .andReturn();
+  }
+
+
+  @Test
+  public void testGetHistoricalPricesEmpty() throws Exception {
```

Comment:
Good starting point, but I love covering all of our cases.
Testing if each value is null, empty, or valid.

Additionally testing our data source. If the IEX server returns a 500 response